### DISCUSSION TITLE
Is there a possibility to set cell margin in flextable and export the output in word

### DISCUSSION THREAD

**Discussion by Kingsman-Key** [davidgohel/flextable#564] [Q&A] [answered]
Our mentor always request us to report our regression table in word and make the result reqdy for publishing. However the process is time consuming. When it comes to wide table, we have to adjust the cell margin to make it fit in the word. I am wonder if it is possible to add a parameter to set cell margin in the flextable to save the trouble of adjusting it every time I put a new table into word.
![image](https://github.com/davidgohel/flextable/assets/60887890/b0c83861-557e-4d5c-8871-d03535c089ce)
The output of left and right cell margin is default to 0.19 centimeter. I tried the **padding** function, But the left and right margin did not change.

**Comment by davidgohel** [comment, marked as answer]
hello, no, it's not a feature
